Salwador/View/ContentView.swift

```diff
@@ -0,0 +1,144 @@
+//
+//  ContentView.swift
+//  Salwador
+//
+//  Created by Ailiniyazi Maimaiti on 31.01.23.
+//
+
+import SwiftUI
+import UIKit
+
+struct ContentView: View {
+    @State private var prompText: String = ""
+    @State private var buttonText: String = "Ready?"
+    @State private var imagePath: String = "salvador-man"
+    @State private var image: UIImage? = nil
+    @State private var isLoading: Bool = false
+
+    var body: some View {
+        ZStack {
+            Color(red: 0.0, green: 0.4666666666666667, blue: 0.7137254901960784)
+
+            VStack {
+                appNameView
+                Spacer()
+                textEditorView
+                submitButtonView
+                Spacer()
+                imageView
+                Spacer()
+            }
+        }
+        .padding()
+    }
+}
+
+struct ContentView_Previews: PreviewProvider {
+    static var previews: some View {
+        ContentView()
+    }
+}
+
+private extension ContentView {
+    var appNameView: some View {
+        VStack {
+            Text("Salvador")
+                .foregroundColor(Color(.white))
+                .font(.title)
+                .fontWeight(.bold)
+                .padding(.top, 30)
+
+            Image("salvador-icon")
+                .resizable()
+                .scaledToFit()
+                .frame(width: 200, height: 100)
+
+            Text("Your AI Image Generator")
+                .foregroundColor(.white)
+                .font(.footnote)
+        }
+    }
+
+    var textEditorView: some View {
+        ZStack {
+            TextEditor(
+                text: $prompText)
+                .background(Color(.red))
+                .font(.title2)
+                .frame(minWidth: 320, idealWidth: 500, maxWidth: .infinity, minHeight: 100, idealHeight: 200, maxHeight: .infinity)
+                .scaledToFit()
+                .border(Color(.black), width: 1)
+                .padding(10)
+                .cornerRadius(10)
+                .onTapGesture {}
+        }
+        .background(Rectangle()
+            .foregroundColor(Color(hue: 0.564, saturation: 0.583, brightness: 0.63))
+            .shadow(radius: 15))
+    }
+
+    var submitButtonView: some View {
+        Button(buttonText) {
+            buttonText = "Wait..."
+            isLoading = true
+            sendRequest(prompText: prompText)
+        }
+        .frame(width: 150, height: 50)
+        .foregroundColor(.white)
+        .background(Color(red: 0.0, green: 0.5882352941176471, blue: 0.7803921568627451))
+        .cornerRadius(10)
+        .padding(10)
```

Comment:
I suggest adding outer paddings not within the subview but at the point when you call it in the main view. So it could be more flexibly reused.
e.g.
```
submitButtonView
    .padding(10)
```